db/migrate/20170805061539_create_mail_logs.rb

```diff
@@ -0,0 +1,15 @@
+class CreateMailLogs < ActiveRecord::Migration
+  def change
+    create_table :mail_logs do |t|
+      t.references :user, index: true
+      t.string :name
+      t.string :email
```

Comment:
@cheeseweasel thanks for explanation. Your point about email being changed in user records is very valid point that i totally missed.